packages/sensors_plus/sensors_plus/CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+## 5.0.0
+
+> Note: This release has breaking changes.
+
+In this release plugin migrated migrated to package:web, meaning that it now supports WASM!
```

Comment:
I think the plugin doesn't depend on package:web at all, instead it uses dart:js_interop directly. It's a small difference, doesn't matter much, just in case you want to mention it.